src/julia-parser.scm

```diff
@@ -1341,7 +1341,7 @@
         (loop (cons (symbol (string.sub (string nxt) 1))
                     path)))
        (else
-        (error (string "invalid \"" word "\" statement")))))))
+        `(,word ,@(reverse path)))))))
```

Comment:
I think this is way too permissive; it allows anything at all to end an import statement, e.g. `import foo (`.